Marlin/example_configurations/K8200/files by Vellemann/Configuration.h

```diff
@@ -0,0 +1,480 @@
+#ifndef CONFIGURATION_H
```

Comment:
> I'm not sure we ought to be adding the configuration from Vellemann here. This will go out of date as Marlin changes, and since we will not be keeping these Configuration files in sync with the rest (including the examples) they will quickly become stale. If you want to keep a historical record of the Vellemann configuration at the present time you should probably create a branch in your Marlin fork that includes these configurations and name it "Vellemann K8200." (If Vellemann decides later to open a Github then you can delete the branch.)

@thinkyhead Thanks again for advice. Now I learned to branch back in time, after finding the point where they branched their version (2013-06-05 !!!) and made a branch from there with their changes committed by me. GitHub is awsome powerfull, but a long way to learn. :-)

See https://github.com/CONSULitAS/Marlin-K8200/tree/Vellemann_firmware_k8200_marlinv2.zip